Cannot find module '../node_modules/lodash/isArray.js'

Hello, I've been using this great library for a while now. I recently tried to fork an existing app and can't get it to work. I'm receiving the following error. Thanks for your work on this library and thanks for taking time to review my issue.

`internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module '../node_modules/lodash/isArray.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (C:\_\myappname\node_modules\cypher-query-builder\dist\cjs5.js:7:32)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)`

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -32,7 +32,7 @@ export default configurations.map(({ target, format }) => {
       babel({ extensions: ['.ts'] }),
     ],
     external: id => dependencies.includes(id)
-      || /\/lodash\//.test(id)
-      || /\/neo4j-driver\//.test(id),
+      || /^lodash/.test(id)
+      || /^neo4j-driver/.test(id),
   };
 });

